### ISSUE TITLE
Add `w3 name get <keyId>` command

### ISSUE THREAD

**Issue by nvrabii** [web3-storage/web3.storage#1835]
## TLDR

I propose adding the `w3 name get <keyId>` command, to retrieve and reuse the signing key associated with an IPNS name. Currently, the revision can be updated only from the environment where the IPNS name was initially created.

I took the initiative and already implemented the solution [here](https://github.com/nvrabii/web3.storage/tree/get-signing-key). If the feature proposal is approved, I'm open to taking further guidance on how to contribute to the project.

## Why add this command?

I was building my website, intending to deploy it on IPFS (Web3.Storage). From an [article](https://betterprogramming.pub/modern-way-to-host-a-site-on-ipfs-7941b65d27c3), I learned how to deploy changes in 3 commands, but I thought I could do better.

When I was trying to update the CID associated with the taken IPNS name via a GitHub action, I could not do that without providing the corresponding signing key. I searched on Internet how to retrieve the key, but I could not find another solution as to modify the `w3`'s code, in order to `console.log` the signing key. Once I retrieved the key, I added it to the repo's secrets, so that my workflow adds the new build to IPFS with a followed update of the IPNS revision.

I don't think that my use case is unique in its kind. Many developers who would like to automate their deployments will have to "move" the signing key from the local to the deployment context.

## Why not add this command?

Maybe there is another way to retrieve the signing key, and this feature is not as useful as I thought.

## Proposed solution

Add the `w3 name get <keyId>` command to the `w3` CLI.

## Code changes

1. Add the command to [w3/bin.js](https://github.com/nvrabii/web3.storage/blob/acb5b7d2a74ba557018a4d3a754f5b9ff73fbafa/packages/w3/bin.js#L72-L75)
2. Add the handler to [w3/name.js](https://github.com/nvrabii/web3.storage/blob/acb5b7d2a74ba557018a4d3a754f5b9ff73fbafa/packages/w3/name.js#L50-L60)
3. Update [w3/README.md](https://github.com/nvrabii/web3.storage/blob/acb5b7d2a74ba557018a4d3a754f5b9ff73fbafa/packages/w3/README.md?plain=1#L90-L94)

## How to test

From what I've seen in the repo, `w3 name ...` commands are [not tested](https://github.com/web3-storage/web3.storage/blob/main/packages/w3/test/bin.spec.js). The code for `Name.getKey(keyId)` reuses the key presence check from [`Name.publish()`](https://github.com/web3-storage/web3.storage/blob/96ff423d778543d0fda4e5f9da6a7ff2417f4587/packages/w3/name.js#L21-L24).

## Risks

There are no reasons to "hide" the signing key in the `config` because it can be easily retrieved by adding a `console.log` either in the `publish` or `create` functions (as a workaround to the existing limitations in the CLI). 

Backward compatibility will be respected because the changes only extend the interface.

**Comment by mbommerez** [comment]
Thanks @nvrabii . We are looking into adding this. We will come back to you.

**Comment by adamalton** [comment]
Thanks, this looks great. I've merged your code into a PR (#2107), and changed the command to `w3 name get-key` to make it clearer what it does.

**Comment by nvrabii** [comment]
@adamalton It makes it clearer indeed. Thank you
